TriangleApp/app/src/main/java/triangle/triangleapp/presentation/impl/SettingsActivity.java

```diff
@@ -0,0 +1,45 @@
+package triangle.triangleapp.presentation.impl;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.preference.PreferenceActivity;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import triangle.triangleapp.R;
+
+/**
+ * Created by D2110175 on 20-6-2017.
+ */
+
+public class SettingsActivity extends PreferenceActivity {
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+
+        super.onCreate(savedInstanceState);
+        addPreferencesFromResource(R.xml.preference);
+
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+
+        menu.add(Menu.NONE, 0, 0, "Show current settings");
+        return super.onCreateOptionsMenu(menu);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+
+        switch (item.getItemId()) {
+            case 0:
```

Comment:
there is only itemID 0, currently, this can be edited to add more menu sections. currently if it isnt id 0 then fall through the switch and let the function return a false. otherwise start intent and return true.